website/content/api-docs/system/auth.mdx

```diff
@@ -99,6 +133,11 @@ For example, enable the "foo" auth method will make it accessible at
   - `allowed_response_headers` `(array: [])` - List of headers to whitelist,
     allowing a plugin to include them in the response.
 
+  - `plugin_version` `(string: "")` – Specifies the semantic version of the plugin
+    to use, e.g. "v1.0.0". If unspecified, the server will select any matching
+    unversioned plugin that may have been registered, the latest versioned plugin
+    registered, or a builtin plugin in that order of precendence.
```

Comment:
nit: I notice we are using a mix of "builtin" and "built-in" in this PR. I grepped the docs and we are also using both however we seem to slightly prefer "built-in". Maybe we should just prefer one or the other in this PR?

Reply:
Good shout, I was trying to write built-in consistently in this PR, but failed in quite a few places 😄